### PR TITLE
1603 containers kubernetes

### LINKED ISSUE #1603: /containers/kubernetes to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /container/kuberneties to vbt
* added .vimeo-container

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/containers/kubernetes)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/4.%20Containers?preview=containers-kubernetes.png) and [demo](http://www.ubuntu.com-1603-containers-kubernetes.demo.haus/containers/kubernetes)

## Issue / Card

Fixes #1603

### CODE REVIEW

**Comment by yaili** [comment]
Design review:
- [ ] Missing external icon link in Watch now, conjure-up, Juju, "Learn more in the Canonical Distribution of Kubernetes datasheet", "Community support"
- [ ] Contact us should be a button - neutral or brand, depends on the importance
- [ ] In the contextual footer, there should be padding bottom in the "Juju for container management" at S screen

File: static/sass/styles-v1.scss [hunk 1]
```diff
@@ -169,6 +171,25 @@ fieldset {
   }
 }
 
+/// container for Vimeo
```
Comment (barrymcgee) [inline static/sass/styles-v1.scss]: Vanilla has 'Embedded media' util for this purpose - https://docs.vanillaframework.io/en/utilities/embedded-media

Comment (pmahnke) [inline static/sass/styles-v1.scss]: thanks!... I missed that

**Comment by pmahnke** [comment]
@yaili fixed

**Comment by yaili** [comment]
@pmahnke you added design +1? 🤔

**Comment by yaili** [comment]
- [ ] Can the image in "Exchange expertise and ideas " be centred in the horizontal space?

**Comment by pmahnke** [comment]
@yaili I fixed that strip, up then updated the copy doc and saw a bunch of copy changes in there... so I made them too, including adding more webinars to that row... hope that looks ok

**Comment by barrymcgee** [comment]
@pmahnke Fix conflicts and all good.

**Pull request merged**